File: POO/temas/2/rb/2.rb
class Frigo
    def abre
        puts "se está abriendo"
    end

    def cierra
        puts "se está cerrando"
    end

    def fija_temperatura
        puts "se cambia la temperatura"
    end
end


mi_frigo = Frigo.new
mi_frigo.abre
mi_frigo.cierra
mi_frigo.fija_temperatura

File: working/temas/c++/2.cpp
#include <iostream>
using namespace std;

class Frigo
{
    public:

    void abre()
    {
        cout << "se está abriendo" << endl;
    }

    void cierra()
    {
        cout << "se está cerrando" << endl;
    }

    void fija_temperatura()
    {
        cout << "se cambia la temperatura" << endl;
    }
};

int main() {
    Frigo mi_frigo;
    mi_frigo.abre();
    mi_frigo.cierra();
    mi_frigo.fija_temperatura();
}
